servicedef/sdk_config.go

```diff
@@ -12,21 +12,20 @@ import (
 )
 
 type SDKConfigParams struct {
-	Credential          string                                      `json:"credential"`
-	StartWaitTimeMS     o.Maybe[ldtime.UnixMillisecondTime]         `json:"startWaitTimeMs,omitempty"`
-	InitCanFail         bool                                        `json:"initCanFail,omitempty"`
-	ServiceEndpoints    o.Maybe[SDKConfigServiceEndpointsParams]    `json:"serviceEndpoints,omitempty"`
-	TLS                 o.Maybe[SDKConfigTLSParams]                 `json:"tls,omitempty"`
-	Streaming           o.Maybe[SDKConfigStreamingParams]           `json:"streaming,omitempty"`
-	Polling             o.Maybe[SDKConfigPollingParams]             `json:"polling,omitempty"`
-	Events              o.Maybe[SDKConfigEventParams]               `json:"events,omitempty"`
-	PersistentDataStore o.Maybe[SDKConfigPersistentDataStoreParams] `json:"persistentDataStore,omitempty"`
-	BigSegments         o.Maybe[SDKConfigBigSegmentsParams]         `json:"bigSegments,omitempty"`
-	Tags                o.Maybe[SDKConfigTagsParams]                `json:"tags,omitempty"`
-	ClientSide          o.Maybe[SDKConfigClientSideParams]          `json:"clientSide,omitempty"`
-	Hooks               o.Maybe[SDKConfigHooksParams]               `json:"hooks,omitempty"`
-	Wrapper             o.Maybe[SDKConfigWrapper]                   `json:"wrapper,omitempty"`
-	DataSystem          o.Maybe[SDKConfigDataSystem]                `json:"dataSystem,omitempty"`
+	Credential           string                                       `json:"credential"`
+	StartWaitTimeMS      o.Maybe[ldtime.UnixMillisecondTime]          `json:"startWaitTimeMs,omitempty"`
+	InitCanFail          bool                                         `json:"initCanFail,omitempty"`
+	ServiceEndpoints     o.Maybe[SDKConfigServiceEndpointsParams]     `json:"serviceEndpoints,omitempty"`
+	TLS                  o.Maybe[SDKConfigTLSParams]                  `json:"tls,omitempty"`
+	Streaming            o.Maybe[SDKConfigStreamingParams]            `json:"streaming,omitempty"`
+	Polling              o.Maybe[SDKConfigPollingParams]              `json:"polling,omitempty"`
+	Events               o.Maybe[SDKConfigEventParams]                `json:"events,omitempty"`
+	BigSegments          o.Maybe[SDKConfigBigSegmentsParams]          `json:"bigSegments,omitempty"`
+	Tags                 o.Maybe[SDKConfigTagsParams]                 `json:"tags,omitempty"`
+	ClientSide           o.Maybe[SDKConfigClientSideParams]           `json:"clientSide,omitempty"`
+	Hooks                o.Maybe[SDKConfigHooksParams]                `json:"hooks,omitempty"`
+	Wrapper              o.Maybe[SDKConfigWrapper]                    `json:"wrapper,omitempty"`
+	PersistenceDataStore o.Maybe[SDKConfigPersistenceDataStoreParams] `json:"persistenceDataStore,omitempty"`
```

Comment:
Ok just one nit. Can we have all the external, SDK facing interfaces use [`PersistentDataStore`](https://docs.launchdarkly.com/sdk/concepts/data-stores) as the terminology. 

I feel like the inconsistency of persistence vs persistent will be one more thing for people to be confused about.

Reply:
Changed it everywhere publicly as requested.